assets/css/_vehicle_map.scss

```diff
@@ -51,4 +51,13 @@
     stroke: $black;
     stroke-width: 1.8;
   }
+
+  .m-vehicle-map-state--auto-centering + .m-vehicle-map & a {
```

Comment:
I noticed that the hover styles are still happening. You might need to actively undo them when they button is disabled.

Reply:
The hover styles seem fixed, but I also changed the cursor to an arrow when the button is disabled.